packages/react/src/components/AccountSettings/DeleteUser/types.ts

```diff
@@ -1,21 +1,33 @@
 import { AmplifyUser } from '@aws-amplify/ui';
+import React from 'react';
+import { ErrorComponent, SubmitButtonComponent } from '../types';
 
-export interface DeleteUserWarningProps {
+export interface WarningProps {
   /** called when end user cancels account deletion */
-  onCancel?: () => void;
+  onCancel: () => void;
   /** called when user acknowledges account deletion */
-  onConfirm?: () => void;
+  onConfirm: () => void;
   /** whether account deletion is in progress */
-  isDisabled?: boolean;
+  isDisabled: boolean;
```

Comment:
Made these required, because parent component always pass those in to the `Warning` components, and are required to function as expected